docs/_data/menu-sql.yaml

```diff
@@ -154,7 +154,9 @@
                   url: sql-ref-syntax-qry-select-distribute-by.html
                 - text: LIMIT Clause 
                   url: sql-ref-syntax-qry-select-limit.html
-                - text: Join Hints
+                - text: JOIN
+                  url: sql-ref-syntax-qry-select-join.html
+                - text: JOIN HINTS
```

Comment:
Why we need to upper case of hints? We don't really have `HINTS` in query.

Reply:
Fixed. Thanks!

Reply:
I changed to ```JOIN hints``` last night, but actually ```JOIN``` is not a SQL keyword here and shouldn't be capitalized.  I will change it back to ```Join Hints```. Sorry for changing this back and forth.